mm2src/coins/z_coin/z_htlc.rs

```diff
@@ -93,7 +102,7 @@ pub async fn z_send_dex_fee(
 }
 
 #[derive(Debug, Display)]
-#[allow(clippy::large_enum_variant, clippy::upper_case_acronyms)]
+#[allow(clippy::large_enum_variant, clippy::upper_case_acronyms, unused)]
```

Comment:
I checked `cargo clippy` for both targets without `unused` and didnt catch the err, I believe you can remove it.

Reply:
that's because `z_coin` is already allowing it